includes/Admin/Settings.php

```diff
@@ -477,6 +477,22 @@ public function get_settings_fields() {
                     'sanitize_callback'          => 'wc_format_decimal',
                     'response_sanitize_callback' => 'wc_format_localized_price',
                 ],
+                'withdraw_order_status' => [
+                    'name'    => 'withdraw_order_status',
+                    'label'   => __( 'Order Status for Withdraw', 'dokan-lite' ),
+                    'desc'    => __( 'Order status for which vendor can make a withdraw request.', 'dokan-lite' ),
+                    'type'    => 'multicheck',
+                    'default' => array(
+                        'wc-completed' => __( 'Completed', 'dokan-lite' ),
+                        'wc-processing' => __( 'Processing', 'dokan-lite' ),
+                        'wc-on-hold' => __( 'On-hold', 'dokan-lite' ),
```

Comment:
can we remove On-hold from default option? I think we can't make withdraw request if order status is on-hold